File: app/src/main/java/br/com/disapps/homepet/ui/common/AppView.java
package br.com.disapps.homepet.ui.common;

import com.hannesdorfmann.mosby3.mvp.MvpView;

/**
 * Created by diefferson.santos on 23/08/17.
 */
public interface AppView extends MvpView, IErrorHandlerView{
    boolean hasInternetConnection();
    void showLoading();
    void dismissLoading();
}

File: app/src/main/java/br/com/disapps/homepet/ui/common/AppView.kt
package br.com.disapps.homepet.ui.common

import com.hannesdorfmann.mosby3.mvp.MvpView

/**
 * Created by diefferson.santos on 23/08/17.
 */
interface AppView : MvpView, IErrorHandlerView {
    fun hasInternetConnection(): Boolean
    fun showLoading(cancelable: Boolean)
    fun dismissLoading()
}
